images/capi/ansible/roles/sysprep/tasks/main.yml

```
@@ -232,3 +232,17 @@
     name: ssh
     enabled: yes
   when: ansible_os_family == "Debian"
+
+- name: Remove temporary builder user
+  ansible.builtin.user:
+    name: builder
+    state: "absent"
+    remove: true
+  when: ansible_os_family == "Ubuntu"
```

```suggestion
  when: ansible_os_family == "Debian"
```

This isn't specific to Ubuntu, is it? It's generally preferable to use "Debian" to cover both distros.

Is this specific to the distro, though? Is there also a `builder` user left around on RHEL, for example?

---

This is not specific for Ubuntu. Fixing it.

---

Would it be better to remove the user in the shutdown command [here](https://github.com/kubernetes-sigs/image-builder/blob/master/images/capi/packer/ova/packer-node.json#L18)